serverless.yml

```diff
@@ -14,7 +14,7 @@ provider:
   region: ${env:AWS_DEFAULT_REGION}
   lambdaHashingVersion: 20201221
   logs:
-    restApi:
+    restApi: # TODO: add data to make the serverless create this role by itself
```

Comment:
We should create an issue for this as well

Reply:
Opened #176 for this